lib/block_view.go

```diff
@@ -6290,6 +6503,11 @@ func (bav *UtxoView) _connectUpdateNFT(
 		return 0, 0, nil, RuleErrorCannotUpdateNonExistentNFT
 	}
 
+	// Verify the NFT is not a pending transfer.
+	if prevNFTEntry.IsPending {
```

Comment:
Do we want to do this check in _connectNFTBid() and _acceptNFTBit() as well or nah?

Reply:
I don't see any harm in making things more strict.  Added.